services/api-server/src/simcore_service_api_server/core/redoc.py

```diff
@@ -32,11 +32,7 @@ def add_vendor_extensions(openapi_schema: dict):
     # TODO: See if openapi-cli does this already
     # TODO: check that all url are available before exposing
     # openapi_schema["paths"][f"/{api_vtag}/meta"]["get"]["x-code-samples"] = [
-    #     {
-    #         "lang": "python",
-    #         "source": {"$ref": f"{PYTHON_CODE_SAMPLES_BASE_URL}/meta/get.py"},
     #     },
-    # ]
 
```

Comment:
why only part of this was removed? Are the comments still relevant?

Reply:
will remove all !